src/System.Runtime/tests/System/Text/RuneTests.netcoreapp.cs

```diff
@@ -550,17 +555,23 @@ public static void TryEncode_AsUtf16(GeneralTestData testData)
             Assert.Equal(testData.Utf16Sequence.Length, charsWritten);
             Assert.True(utf16Buffer.SequenceEqual(testData.Utf16Sequence));
 
+            utf16Buffer.Clear();
+            Assert.Equal(testData.Utf16Sequence.Length, rune.Encode(utf16Buffer));
+            Assert.True(utf16Buffer.SequenceEqual(testData.Utf16Sequence));
+
             // Finally, try with a buffer that's too long (should succeed)
 
             utf16Buffer = stackalloc char[rune.Utf16SequenceLength + 1];
             success = rune.TryEncode(utf16Buffer, out charsWritten);
             Assert.True(success);
             Assert.Equal(testData.Utf16Sequence.Length, charsWritten);
             Assert.True(utf16Buffer.Slice(0, testData.Utf16Sequence.Length).SequenceEqual(testData.Utf16Sequence));
+
+            utf16Buffer.Clear();
+            Assert.Equal(testData.Utf16Sequence.Length, rune.Encode(utf16Buffer));
+            Assert.True(utf16Buffer.Slice(0, testData.Utf16Sequence.Length).SequenceEqual(testData.Utf16Sequence));
         }
 
-        // This test needs use private reflection into Rune at the moment because the
-        // TryEncode method which writes as UTF-8 isn't yet public.
```

Comment:
This comment was just stale, right?  Asking because I don't see reflection being used or removed along with the deletion.

Reply:
Yeah, it was a stale comment.